test/system/freertos/sleep.c

```diff
@@ -18,14 +18,14 @@
 static int sleep(void)
 {
 	int rc;
-	unsigned int usec = 3;
+	unsigned int usec = 10000;
```

Comment:
This value corresponds to your  portTICK_PERIOD_MS, right?
I would suggest to use a non aligned value.

Reply:
Changed in the updated commit.